README.md

```diff
@@ -82,7 +82,7 @@ Note that the `edxapp` user does not have sudo rights, so all following commands
 Create folder in which everything will be installed:
 
     sudo mkdir /opt/openedx
-    sudo chown edxapp:edxapp /opt/openedx
+    sudo chown edxapp:www-data /opt/openedx
```

Comment:
Let's play it safe and keep `edxapp:edxapp` for now. It conveys the information that no additional user needs write access to the folder. If users have more experience, like you, then they can decide which group to assign by themselves.